iphone/Maps/Maps.xcodeproj/project.pbxproj

```diff
@@ -3696,6 +3742,11 @@
 							};
 						};
 					};
+					ED097E752BB80C320006ED01 = {
+						CreatedOnToolsVersion = 15.2;
```

Comment:
Is it the latest Xcode?

Reply:
No. Latest is 15.3. Should this line be removed?